src/Arriba/Arriba.Test/Serialization/CSV/CsvReaderTests.cs

```diff
@@ -227,7 +227,7 @@ private static MemoryStream StreamFromString(string content)
 
             using (StreamWriter writer = new StreamWriter(ms, Encoding.UTF8, 1024, true))
             {
-                foreach (var line in content.Split(Environment.NewLine))
+                foreach (var line in content.Replace("\r\n", "\n").Split("\n"))
```

Comment:
Is this for the fact that the crawler my me executed in different OS?

Reply:
Split on Environment.NewLine works if every file on that OS is consistent. However I for instance use both WSL and Windows so I had had files with both \r\n and \n which will cause test failures. Ideally this should be abstracted away, but didn't want to do that as part of this change. I mainly was trying to make minimal edits to keep PR focus on moving and breaking code up